guide/popular-topics/audit-logs.md

```diff
@@ -0,0 +1,167 @@
+# Working with Audit Logs
+
+## Some quick background
+Audit logs are a great moderation tool offered by discord to know what happened in a server, and usually by whom. There are quite a few cases where audit logs may be used, this guide will limit itself to the most common use cases. For more information on audit logs, feel free to consult the [relevant discord api page](https://discordapp.com/developers/docs/resources/audit-log).
```

Comment:
I'd suggest rephrasing this section and put emphasis on the fact it is the only way to find out who the executor of moderative actions is and that events do not provide that information, hence making fetches necessary